src/context/AppProvider/AppContext.tsx

```diff
@@ -287,46 +283,16 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
     })()
   }, [dispatch, requestedAccountIds, portfolioLoadingStatus])
 
-  const uniV2LpIdsData = useAppSelector(
-    opportunitiesApi.endpoints.getOpportunityIds.select({
-      defiType: DefiType.LiquidityPool,
-      defiProvider: DefiProvider.UniV2,
-    }),
-  )
-
   const marketDataPollingInterval = 60 * 15 * 1000 // refetch data every 15 minutes
   useQuery({
     queryKey: ['marketData', {}],
     queryFn: async () => {
-      // Exclude assets for which we are unable to get market data
-      // We would fire query thunks / XHRs that would slow down the app
-      // We insert price data for these as resolver-level, and are unable to get market data
-      const excluded: AssetId[] = uniV2LpIdsData.data ?? []
-      const portfolioAssetIdsExcludeNoMarketData = difference(portfolioAssetIds, excluded)
-
-      // Only commented out to make it clear we do NOT want to use RTK options here
-      // We use react-query as a wrapper because it allows us to disable refetch for background tabs
-      // const opts = { subscriptionOptions: { pollingInterval: marketDataPollingInterval } }
-      const timeframe = DEFAULT_HISTORY_TIMEFRAME
-
-      await Promise.all([
-        dispatch(
-          marketApi.endpoints.findPriceHistoryByAssetIds.initiate(
-            {
-              timeframe,
-              assetIds: portfolioAssetIdsExcludeNoMarketData,
-            },
-            // Since we use react-query as a polling wrapper, every initiate call *is* a force refetch here
-            { forceRefetch: true },
-          ),
-        ),
-        dispatch(
-          marketApi.endpoints.findByAssetIds.initiate(portfolioAssetIdsExcludeNoMarketData, {
-            // Since we use react-query as a polling wrapper, every initiate call *is* a force refetch here
-            forceRefetch: true,
-          }),
-        ),
-      ])
+      await dispatch(
+        marketApi.endpoints.findByAssetIds.initiate(portfolioAssetIds, {
+          // Since we use react-query as a polling wrapper, every initiate call *is* a force refetch here
+          forceRefetch: true,
+        }),
+      )
```

Comment:
Kept spot data fetching here, but removed historical market-data fetching, now moved to `useFetchPriceHistories()` which is a saner (only consumed in Dashboard and asset page), and more optimized (only fetching either the current AssetId for asset page, and AssetIds above threshold for dashboard).